### DISCUSSION TITLE
LICENSE to reflect the V4 licence changes ?

### DISCUSSION THREAD

**Discussion by andy778** [serverless/serverless#12884] [General]
LICENSE is today in the github master MIT,  but shouldn't it reflect the new licence ?

**Comment by jormaechea** [comment]
As per the README,

> Please note, the structure and licensing of the V.4 repository differ from the V.4 npm module. The npm module contains some proprietary licensed software, as V.4 transitions to a common SaaS product, [as previously announced](https://www.serverless.com/blog/serverless-framework-v4-a-new-model). The original Serverless Framework source code and more will continue to remain MIT license software, the repository will soon be restructured to clearly distinguish between proprietary and open-source components.

As I understand, the license change will apply to the npm module, but not to the git repository? I suppose that this public repo is a part of a greater tool that ends up published in npm, which includes proprietary code and cloud licensing checks..

Does this mean that the use of the code in the repository (and all upcoming updates to it) will remain free for everything stated in the [MIT License](https://github.com/serverless/serverless/blob/16e3ca1b369440b384f76ff4926bf6b23a6abc0c/LICENSE)?

Maybe someone from Serverless Inc. can confirm?

Is there an ETA for the following change?
> the repository will soon be restructured to clearly distinguish between proprietary and open-source components
